### ISSUE TITLE
Raspberry Pi 4 8GB ModuleNotFoundError: No module named 'PyViCare.PyViCareDevice'

### ISSUE THREAD

**Issue by julred** [openviess/PyViCare#81]
Dear all,

I'm having problems to run PyViCare script on my Raspberry Pi 4 8GB (64-Bit OS):

```
pi@raspberrypi:~/Skripte $ uname -a
Linux raspberrypi 5.10.17-v8+ #1403 SMP PREEMPT Mon Feb 22 11:37:54 GMT 2021 aarch64 GNU/Linux
```

I tried to use the basic test command (of course, I changed it to my credentials):
```
python3 -c "from PyViCare.PyViCareDevice import Device; from PyViCare.PyViCareGazBoiler import GazBoiler; t=GazBoiler('youremail@domain.xxx','yourVicarepassword','/tmp/vicare_token.save');print(t.getPrograms())"
```

This was the response:
```
Traceback (most recent call last):
  File "<string>", line 1, in <module>
ModuleNotFoundError: No module named 'PyViCare.PyViCareDevice'
```

The script is working also on a Windows 10 64 Bit machine.
What might be the problem?

// Edit;
Just double checked via `pip show pyvicare` the installed module version.
On Windows 10 I'll get the V 0.2.5 which is the latest release.
On Raspberry Pi I'll get the V 0.11... But I installed both on the same date.

**Comment by julred** [comment]
I installed V0.2.5 via .whl file.

```
pi@raspberrypi:~/Skripte $ pip show pyvicare
Name: PyViCare
Version: 0.2.5
Summary: Library to communicate with the Viessmann ViCare API
Home-page: https://github.com/somm15/PyViCare
Author: Simon Gillet
Author-email: mail+pyvicare@gillet.ninja
License: UNKNOWN
Location: /home/pi/.local/lib/python3.7/site-packages
Requires: requests-oauthlib, simplejson
Required-by:
```

But still the same error log as mentioned above:
```
pi@raspberrypi:~/Skripte $ python vicaretest.py
Traceback (most recent call last):
  File "vicaretest.py", line 4, in <module>
    from PyViCare.PyViCareGazBoiler import GazBoiler
ImportError: No module named PyViCare.PyViCareGazBoiler

```

**Comment by julred** [comment]
Any ideas?

**Comment by SchwarzJJ** [comment]
I have a similar problem with python3 on raspi4:
```
pi@raspi4:~ $ python3 pyvicare2openhab.py 
Traceback (most recent call last):
  File "pyvicare2openhab.py", line 9, in <module>
    from PyViCare.PyViCareGazBoiler import GazBoiler
ModuleNotFoundError: No module named 'PyViCare.PyViCareGazBoiler'
```
```
pi@raspi4:~ $ pip3 show pyvicare
Name: PyViCare
Version: 0.11
Summary: Library to communicate with the Viessmann ViCare API
Home-page: https://github.com/somm15/PyViCare
Author: Simon Gillet
Author-email: mail+pyvicare@gillet.ninja
License: UNKNOWN
Location: /home/pi/.local/lib/python3.7/site-packages
Requires: requests-oauthlib
Required-by: 
```
```
#!/usr/bin/python3

import argparse
import sys
import logging
import requests
from requests import Session
from PyViCare.PyViCareGazBoiler import GazBoiler
```

Any ideas?

**Comment by galaxis1309** [comment]
Hi,
same issue, is there a step-by-step install guide available ? Perhaps i'm doing things wrong using pip "install pyvicare" resulting in v0.2.5 via pip show ??

**Comment by woehrl01** [comment]
Hi, I just published 1.0.0 to PyPi. Please validate if it's working for you
